state-chain/cf-integration-tests/src/swapping.rs

```diff
@@ -391,7 +396,11 @@ fn can_process_ccm_via_swap_deposit_address() {
 			Some(message),
 			0u16,
 			Default::default(),
-			None,
+			ChannelRefundParametersEncoded {
+				retry_duration: 0,
+				refund_address: EncodedAddress::Eth([1u8; 20]),
+				min_price: sp_core::U256::zero(),
```

Comment:
Same here, we could use the already-defined constant instead of repeating the same thing.